src/engraving/libmscore/chord.cpp

```diff
@@ -1491,7 +1491,7 @@ qreal Chord::calcDefaultStemLength()
     _minStemLength = calcMinStemLength();
     qreal minStemLengthSpaces = _minStemLength / _spatium;
 
-    int staffLineCount = staffItem->lines(tick());
+    int staffLineCount = staffItem ? staffItem->lines(tick()) : 4;
```

Comment:
After #9886 is merged, you can probably also write 
```c++
int staffLineCount = staffType ? staffType->lines() : 4;
```
which eliminates is more efficient.

I also think that a default value of `5` instead of `4` would make more sense musically.